### PR TITLE
docs(instance): ipmob multiple IPs or IPv6 only examples

### PR DESCRIPTION
**Pull request by ziirish**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Add examples regarding the use of the new IP Mobility feature with IPv6 only and multiple IPs

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 1]
```diff
@@ -239,6 +237,350 @@ Run the following command to create an Instance with the previously created IP a
     </TabsTab>
 </Tabs>
 
+## Create a new Instance with one IPv6 only
+
+First you need to create an IPv6 reservation, then you create a new IP Mobility Instance with this IP attached.
```
Comment (RoRoJ) [inline compute/instances/api-cli/using-ip-mobility.mdx:242]:
```suggestion
First you need to create an IPv6 reservation, then you create a new IP mobility Instance with this IP attached.
```

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 2]
```diff
@@ -239,6 +237,350 @@ Run the following command to create an Instance with the previously created IP a
...
+                    }
+                },
+                "zone": "pl-waw-2"
+            }
+        }
+        ```
+        <Message type="tip">
+            You will find the configuration of your IPs within the `public_ips` key. Note that IPs with a `provisioning_mode` in `DHCP` or `SLAAC`
+            are automatically configured, whereas the `manual` might need manual configuration unless you are using one of our supported Images OS
+            in which case `cloud-init` should take care of any extra IP at boot time.
```
Comment (RoRoJ) [inline compute/instances/api-cli/using-ip-mobility.mdx:579]:
```suggestion
            in which case `cloud-init` should take care of any extra IPs at boot time.
```

File: compute/instances/api-cli/using-ip-mobility.mdx (hunk 1 above)
Comment (ziirish) [reply compute/instances/api-cli/using-ip-mobility.mdx:242]: other occurrences of this in the page are written `IP Mobility` like [here](https://github.com/scaleway/docs-content/blob/c2cb623e3dd883a74ac1e6dd1373ec4c18fad2e7/compute/instances/api-cli/using-ip-mobility.mdx?plain=1#L30) and the following lines.

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 3]
```diff
@@ -239,6 +237,350 @@ Run the following command to create an Instance with the previously created IP a
...
+                        "zone": "pl-waw-2"
+                    }
+                },
+                "zone": "pl-waw-2"
+            }
+        }
+        ```
+        <Message type="tip">
+            You will find the configuration of your IPs within the `public_ips` key. Note that IPs with a `provisioning_mode` in `DHCP` or `SLAAC`
+            are automatically configured, whereas the `manual` might need manual configuration unless you are using one of our supported Images OS
```
[the remark this answers is not in the document]
Comment (ziirish) [reply compute/instances/api-cli/using-ip-mobility.mdx]: updated

Thanks

File: compute/instances/api-cli/using-ip-mobility.mdx (hunk 2 above)
Comment (ziirish) [reply compute/instances/api-cli/using-ip-mobility.mdx]: updated

Thanks

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**